nautobot/docs/user-guide/platform-functionality/webhook.md

```diff
@@ -1,135 +1,119 @@
+- **HTTP method**: `POST`
+- **URL**: Slack incoming webhook URL
+- **HTTP content type**: `application/json`
+- **Body template**:
+  
+  ```json
+  {"text": "IP address {{ data['address'] }} was created by {{ username }}!"}
+  ```
 
-The following data is available as context for Jinja2 templates:
+### Available Context Variables
 
-* `event` - The type of event which triggered the webhook: created, updated, or deleted.
-* `model` - The Nautobot model which triggered the change.
-* `timestamp` - The time at which the event occurred (in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format).
-* `username` - The name of the user account associated with the change.
-* `request_id` - The unique request ID. This may be used to correlate multiple changes associated with a single request.
-* `data` - A serialized representation of the object _after_ the change was made. This is typically equivalent to the model's representation in Nautobot's REST API.
-* `snapshots` - snapshots of the serialized object state both before and after the change was made; provided as a dictionary with keys named `prechange`, `postchange` and `differences`.
+| Variable | Description |
+|----------|-------------|
+| `event` | The event type (`created`, `updated`, `deleted`). |
+| `model` | The Nautobot model triggering the event. |
+| `timestamp` | The event timestamp in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format. |
+| `username` | The user who triggered the event. |
+| `request_id` | A unique request ID for correlation of multiple changes associated with a single request. |
+| `data` | A serialized representation of the object *after* the change. |
+| `snapshots` | Contains `prechange`, `postchange`, and `differences` snapshots. |
 
-### Default Request Body
+## Default Request Body
 
-If no body template is specified, the request body will be populated with a JSON object containing the context data. For example, a newly created Location might appear as follows:
+If no body template is provided, Nautobot sends a default JSON payload:
 
-```no-highlight
+```json
 {
     "event": "created",
-    "timestamp": "2023-02-14 12:34:56.000000+00:00",
+    "timestamp": "2023-02-14T12:34:56.000000+00:00",
     "model": "location",
     "username": "admin",
     "request_id": "fab0a4fb-52ba-4cb4-9756-4e6a3ac05332",
-    "data": {
-        "id": "5e4f9a91-372b-46df-a50a-c26357475bee",
-        "display": "Campus A",
-        "url": "/api/dcim/locations/5e4f9a91-372b-46df-a50a-c26357475bee/",
-        "name": "Campus A",
-        "status": {
-            "display": "Active",
-            "id": "363a431c-c784-40b5-8513-758cafd174ad",
-            "url": "/api/extras/statuses/363a431c-c784-40b5-8513-758cafd174ad/",
-            "name": "Active",
-            "created": "2023-02-14T00:00:00Z",
-            "last_updated": "2023-02-14T19:40:13.216150Z"
-        },
-        ...
-    },
+    "data": { "id": "5e4f9a91-372b-46df-a50a-c26357475bee", "name": "Campus A" },
```

Comment:
Is this change accurate? I would expect a `created` snapshot to include the full data (as in the `before` diffs), not just the `id` and the `name`.